proxy/src/main/java/com/velocitypowered/proxy/Velocity.java

```diff
@@ -54,14 +60,23 @@ public static void main(String... args) {
 
     long startTime = System.currentTimeMillis();
 
-    VelocityServer server = new VelocityServer(options);
+    VelocityPluginManager pluginManager;
+    VelocityEventManager eventManager;
+    VelocityConsoleCommandSource console;
+    VelocityServer server = new VelocityServer(
+            (pluginManager = new VelocityPluginManager()),
```

Comment:
Can you move these assignments to the variables directly, and reference them normally here instead?

Reply:
It will be less beautiful then. I will do it, though